### ISSUE TITLE
Version 1.0.18 crashes trying to open epubs generated by wikisource.org

### ISSUE THREAD

**Issue by monich** [monich/harbour-books#21]
Such as this one: https://de.wikisource.org/wiki/Die_Edda_(Simrock_1876)

**Comment by monich** [comment]
It dies trying to parse this inline CSS:
```
"border-spacing:0; padding:4; border-width:2; margin:0em 0em 1em 0em; border:solid 1px #AAAAAA; border-collapse:collapse; background-color:#F9F9F9; font-size:95%; empty-cells:show; color:black; {{{1}}}"
```
Particularly it doesn't like the `{{{1}}}` part. I'm not sure what that's supposed to mean but it should be easy enough to ignore it.

**Comment by monich** [comment]
Fixed by commit 83ca0db

**Comment by monich** [comment]
Released in version 1.0.19
